src/components/ExperienceTable/ExperienceTable.tsx

```diff
@@ -34,30 +34,27 @@ export const ExperienceTable = ({
     <div className="overflow-x-auto">
       {rows.length > 0 && (
         <table className="w-full" border={0} cellPadding={0} cellSpacing={0}>
-          <thead>
-            <tr className="border-b border-inactive-gray text-left text-body-md">
-              <th scope="col" className="font-normal pr-7 pb-3">
+          <thead className="after:content-[''] after:block after:h-5">
+            <tr className="border-b border-inactive-gray text-left text-body-md font-normal">
```

Comment:
`font-normal` shoud not be needed as the `text-body-md` tolds the same weight already.

But it seems that `th` got some `user agent stylesheet` that bolds the text in `th`. To still have  weight as 400, it seems to be necessary to told `font-normal` for the `th`s